package-system/assimp/install_assimp_linux.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Source_Subfolder": "src",
+  "Install_Paths": [
+    "include/",
+    "lib/libassimp.a",
+    "bin/assimp",
+    "bin/libassimp.so.5.0.1",
+    "bin/unit",
```

Comment:
Where does this come from?  is it something that assimp makes?

Reply:
I meant specifically the 'unit' executable.

Reply:
yeah, that's the unit test executable. Previously we weren't including it in the package, but I decided to put it in because I thought it would be useful when debugging tougher problems with assimp. Running the unit test executable locally can server as a good step to verify a baseline of functionality is working correctly.

I've also set it to run in the test step, so it's run as part of this packaging work.

Reply:
Definitely good to run the tests that are included in the package but an arguably more important test is one that uses the package itself.   You could save yourself a lot of worry and trouble that way since it short cuts the iteration cycle of ingesting this package in O3DE.

What I mean by a test that uses the package itself is, e.g. a separate folder in your 3p-package-source location (like a subfolder called test or something). Which contains a main.cpp and a CMakeLists.txt that declares an executable and calls find_package(ASSIMP).   As long as the _final output folder_ of the package (plus where zlib was dropped) is in the CMAKE_MODULE_PATH, it should work, and it should work identically to how its likely to work in the O3DE side, revealing any linkage errors or include problems.

Reply:
Good idea, I've added it to the list of follow up work I want to do after getting the first part of this done.